contributing.md

```diff
@@ -79,88 +125,67 @@ $ git config --global user.name "J. Random User"
 $ git config --global user.email "j.random.user@example.com"
 ```
 
-Writing good commit logs is important.  A commit log should describe what
-changed and why.  Follow these guidelines when writing one:
+Writing good commit logs is important.  A commit log should describe what changed and why.  Follow these guidelines when writing one:
 
-1. The first line should be 50 characters or less and contain a short
-   description of the change prefixed with the name of the changed
-   subsystem (e.g. "net: add localAddress and localPort to Socket").
-2. Keep the second line blank.
-3. Wrap all other lines at 72 columns.
+- 50 characters or less 
```

Comment:
If this isn't enforced by a tool, I don't think its worth putting here.

Reply:
More of a general guideline, but agreed it's a bit arbitrary. Removing.